### DISCUSSION TITLE
NCA on PK collected at steady state

### DISCUSSION THREAD

**Discussion by kylebaron** [humanpred/pknca#321] [Q&A] [answered]
Hi @billdenney  - 

Long time listener, first time caller. Is there a way to tell PKNCA that I think the concentrations being analyzed got collected at steady state? I have PK collected after first dose and after a dose presumably given at steady state and concentrations covering the dosing interval (out to tau after the dose).  I want AUC,infinity to be AUC,tau in that case. Or am I not doing this correctly?

Thanks!
Kyle

**Comment by billdenney** [comment]
I think you're looking for how to setup multiple intervals: https://billdenney.github.io/pknca/articles/v03-selection-of-calculation-intervals.html#multiple-intervals

You'd setup one interval for the single-dose case an one for steady-state. That would look something like:

```
d_intervals <-
  data.frame(start = c(0, 168), end = 24+c(0, 168), aucint.inf.obs = TRUE, cmax = TRUE, tmax = TRUE) # add any other parameters you want
```

With the above and that pointer to the documentation, does that answer your question?

**Comment by kylebaron** [comment]
Thanks for the quick reply, @billdenney ; this is really helpful and I believe it answers the original question. I had been manually splitting the data and calling twice; but this looks better. 

If I could ask one more: If I'm in this situation: 

<img width="653" alt="Screenshot 2024-10-18 at 06 51 21" src="https://github.com/user-attachments/assets/efc30c57-9fbe-443c-9b96-96ba5fb1dff5">

and I want to get clearance calculated from the data in green shading, is that just a matter of picking the right clearance output? 

<img width="815" alt="Screenshot 2024-10-18 at 06 57 40" src="https://github.com/user-attachments/assets/6247298b-c9fc-4c41-a814-754bfae4bb2d">


Thanks again for your help!

Kyle

**Reply by billdenney** [reply, marked as answer]
Yes, one of the benefits of PKNCA is that you can do whatever NCA you need to do with a single pass. You never need to split the data. (If you ever find an exception to this, please let me know.)

Intervals can be overlapping, too. For instance, if you want to calculate NCA parameters for a whole day and partial AUCs, you can do both at once.

Yes, it should be a matter of choosing the correct clearance.
